ztp/source-crs/extra-manifest/disk-encryption.yaml.tmpl

```diff
@@ -14,18 +14,38 @@ spec:
         - name: root
           device: /dev/disk/by-partlabel/root
           clevis:
+          {{- if and (eq .Type "tpm2") (ne .Tpm2.PCRList "") }}
+            custom:
+              pin: "tpm2"
+              config: '{"t":1,"pins":{"tpm2":[{"hash":"sha256","key":"ecc","pcr_bank":"sha256","pcr_ids":"{{ .Tpm2.PCRList }}"}]}}'
+          {{- else if and (eq .Type "tpm2") (eq .Tpm2.PCRList "") }}
+            tpm2: true
+          {{- else if eq .Type "nbde" }}
             tang:
             {{- range .Tang }}
                 - url: "{{ .URL }}"
                   thumbprint: "{{ .Thumbprint }}"
             {{- end }}
+          {{- else if eq .Type "tpm2+nbde" }}
+            custom:
+              pin: "sss"
+              needsNetwork: true
+              config: '{"t":2,"pins":{"tpm2":[{}],"sss":[{"t":1,"pins":{"tang":[
```

Comment:
This should probably also add PRS at if .Tmp2.PCRList is set, right?

Reply:
removed support for nbde+tpm2 for this first PR